lib/messages/flow/signout.js

```diff
@@ -0,0 +1,28 @@
+const { Message } = require('..');
+
+module.exports = class SignOut extends Message {
+  constructor(signInLink, userSlackId) {
+    super({});
+    this.signInLink = signInLink;
+    this.userSlackId = userSlackId;
+  }
+
+  toJSON() {
+    return {
+      response_type: 'ephemeral',
+      attachments: [{
+        ...this.getBaseMessage(),
+        fallback: 'Your GitHub account has been disconnected.',
+        text: `:white_check_mark: <@${this.userSlackId}> is disconnected from your GitHub account\n` +
+          'Features like subscriptions and unfurls will stop working. ' +
+          'To re-connect your GitHub account click the button below.',
```

Comment:
I know we're already using  "connect" elsewhere, but what about switching to the language of "signed in" and "signed out". I also don't think we've been using "unfurls" in any user-facing language.

> :white_check_mark: <@U98765> is now signed out. Use `/github signin` to sign back in to your GitHub account at any time.

🤷‍♂️ 

@cmwinters, what do you think?

Reply:
@bkeepers seems fine to me.

Reply:
👍 Also prefer to avoid "connect". I think it is valuable to tell the user that features will stop working though. That might not be obvious to them.